static/gsApp/components/gsBanner.tsx

```diff
@@ -135,7 +135,7 @@ function SuspensionModal({Header, Body, Footer, subscription}: SuspensionModalPr
       <Footer>
         <ZendeskLink
           subject="Account Suspension"
-          className="btn btn-primary"
+          Component={props => <LinkButton {...props} href={props.href ?? ''} />}
```

Comment:
Guessing the `props.href ?? ''` is just to appease TypeScript? Should we use a guard that throws instead?